artifacts/camara-cloudevents/event-subscription-template.yaml

```diff
@@ -864,17 +890,13 @@ components:
           schema:
             $ref: "#/components/schemas/ErrorInfo"
           examples:
-            PermissionDenied:
-              value:
-                status: 403
-                code: "PERMISSION_DENIED"
-                message: "Client does not have sufficient permissions to perform this action"
-            TokenMismatch:
+            GENERIC_403_PERMISSION_DENIED:
+              description: Permission denied. OAuth2 token access does not have the required scope or when the user fails operational security
               value:
                 status: 403
-                code: "SUBSCRIPTION_MISMATCH"
-                message: "Inconsistent access token for requested events subscription"
-    Generic403:
+                code: PERMISSION_DENIED
+                message: Client does not have sufficient permissions to perform this action.
+    CreateSubscriptionBadRequest403:
```

Comment:
Here it lacks as an option (examples):
            GENERIC_403_SUBSCRIPTION_MISMATCH:
              value:
                status: 403
                code: "SUBSCRIPTION_MISMATCH"
                message: "Inconsistent access token for requested events subscription"

Reply:
Yes make sense to add this exemple for the generic - I add it.